gems/active_decorator/1.4/manifest.yaml

```diff
@@ -0,0 +1,7 @@
+# manifest.yaml describes dependencies which do not appear in the gemspec.
+# If this gem includes such dependencies, comment-out the following lines and
+# declare the dependencies.
+# If all dependencies appear in the gemspec, you can remove this file.
```

Comment:
Could you remove this file? It looks not necessary.
`manifest.yaml` is needed to list standard libs dependencies, and the `active_decorator.rbs` doesn't have the dependencies. So it is not necessary.

Reply:
Thanks for your review!
I removed `manifest.yaml’ file.
I missed comments in codes😅